tox.ini

```diff
@@ -40,6 +41,15 @@ deps =
     flake8==3.7.9
 commands = flake8 datadog
 
+[testenv:mypy]
+basepython = python3
```

Comment:
Mypy must be run in Python 3.5 or higher (but it can still type-check Python 2 code).

Reply:
I added a comment to the file for future reference.